### PR TITLE
Make sure nexus is used

### PR DESCRIPTION
**Pull request by asmigala**
//skip-ci

### CODE REVIEW

File: utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java [hunk 1]
```diff
@@ -323,9 +325,11 @@ private static void fixMavenRepos() {
                 replacementRepo = TestConfiguration.upstreamRepository();
             } else {
                 // no replacement, will use maven central
+                log.warn("no replacement repo, skipping");
                 return;
             }
         }
+        log.info("trying to fix maven repo with {}", replacementRepo);
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java]: - please start the logs with uppercase
- I would rephrase this - I know the method is called `fix` maven repos and it sounds like working around a bug - plain `Replacing maven repos with..` sounds better

Comment (asmigala) [reply utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java]: done both

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED
